src/OpenConext/ProfileBundle/Storage/SingleCookieStorage.php

```diff
@@ -42,13 +43,43 @@ class SingleCookieStorage implements EventSubscriberInterface
      */
     private $cookieValue;
 
-    public function __construct($cookieDomain, $cookieKey)
-    {
+    /**
+     * @var DateTime|0
+     */
+    private $cookieExpirationDate;
+
+    /**
+     * @var boolean
+     */
+    private $cookieSecure;
+
+    /**
+     * @var boolean
+     */
+    private $cookieHttpOnly;
+
+    public function __construct(
+        $cookieDomain,
+        $cookieKey,
+        DateTime $cookieExpirationDate = null,
+        $cookieSecure = true,
+        $cookieHttpOnly = true
```

Comment:
These defaults do not match the defaults of the configuration, perhaps make them the same?